backend/main.py

```diff
@@ -145,7 +145,8 @@ def add_new(news_data):
     session.commit()
     session.close()
 
-
+news_channelId = 2
+first_page = 1
```

Comment:
The magic number is usually not the dynamic variable. According to PEP 8, the constant variable should defined on a module level and written in all capital letters with underscores separating words

[reference](https://peps.python.org/pep-0008/#constants)